plugins/ecc-vscc/ecc_validation_logic.go

```diff
@@ -121,70 +120,70 @@ func (vscc *VSCCECC) checkEnclaveEndorsement(cis *peer.ChaincodeInvocationSpec,
 		return err
 	}
 
-	for _, ns := range txRWSet.NsRwSets {
-		logger.Debugf("Namespace %s", ns.NameSpace)
-
-		// TODO make this more flexible
-		if !strings.HasPrefix(ns.NameSpace, "ecc") {
-			continue
-		}
```

Comment:
I guess this we dropped as, as discussed, by definition the ecc plugin should be configured exactly (and only) for ecc transactions?

Reply:
yes